templates/accounts/resend_verification_email.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <img src="{% static 'accounts/icon/mail-sent.gif' %}" alt="Email Sent" style="max-width: 200px;">
```

Comment:
This icon doesn't fit here, as the same icon is shown on the next page. See if you can find a more suitable icon for this page, otherwise you can leave it without any icon.